submariner/ipsec-cert-issuer.md

```diff
@@ -0,0 +1,114 @@
+# IPSec Certificate Issuer
+
+## Summary
+
+Submariners current security arrangements, namely using an identical pre-shared key in
+all deployments will fail security assessment in most environments which have to comply
+with a pre-existing security policy and rules on using VPNs in general and IPSEC in particular.
+
+In order to comply the minimum enhancement that needs to be implemented is the use
```

Comment:
The alternative is to ensure we use one PSK per connected cluster. This would require inventing our own ways of authenticating clients and revocating PSKs; CAs provide standardised ways of handling all this.

Reply:
I think it would be useful to capture this in the EP